scripts/prompt-config.sh

```diff
@@ -828,6 +828,16 @@ if [[ -n ${answer_file} ]]; then
 	fi
 elif [[ -f ${USBMNT}/private/answers.json ]]; then
 	answer_file=${USBMNT}/private/answers.json
+elif [[ -f /system/boot/tinkerbell.json ]]; then
+	tmp_answers=$(mktemp)
+	"${USBMNT}/scripts/tinkerbell2answers.sh" /system/boot/tinkerbell.json \
+		> "$tmp_answers"
+	# We're going make a quick sanity test of the candidate answers file
+	# to see if it's usable, but we're not going to exhaustively lint it.
```

Comment:
Probably wanna guard against /31, but other than that, cool!

Reply:
I'm not sure there's anything worthwhile doing here. It's basically not going to work if it's a /31. Even if we did cause a fail or log an error somewhere, I'm not sure a user would reasonably be expected to find it. I don't really know how to resolve that.

This is also intended (though, admittedly, not required) to be used in conjunction with my other EQM scripts, which will automatically reserve a /28 for each CN.

In any case, the requirements will need to be well documented (e.g., vlans, external network size).